INSTALL.md

```diff
@@ -3,6 +3,7 @@
 There are 3 steps to follow under **Installation Instructions**.
```

Comment:
this is item 4... :/

I'm hesitant to put this upfront in the installation instructions: there's a lot for new users to understand here, and I'm not entirely sure it belongs here

Reply:
maybe link it from the README instead?

Reply:
Fair, its a bit of a tricky balance. Maybe we should point to the doc whenever we mention min requirements?